src/internal/kopia/wrapper.go

```diff
@@ -190,6 +192,19 @@ func (w Wrapper) ConsumeBackupCollections(
 		return nil, nil, nil, clues.Wrap(err, "building kopia directories")
 	}
 
+	// Add some extra tags so we can look things up by reason.
```

Comment:
🎉

Reply:
technically this doesn't add anything extra from what we had previously. It just ensures things keep working but is commented a bit better